miqa/core/rest/experiment.py

```diff
@@ -115,20 +115,24 @@ def lock(self, request, pk=None):
             experiment: Experiment = Experiment.objects.select_for_update().get(pk=pk)
             if experiment.project.archived:
                 raise ArchivedProject()
-            if experiment.lock_owner is not None and experiment.lock_owner != request.user:
+            if (
+                experiment.lock_owner is not None
+                and experiment.lock_owner != request.user
+                and not (request.user.is_superuser and 'force' in request.data)
```

Comment:
This doesn't actually check the value of `force`, so if you included `force: False` it would count as being a forced lock.

Reply:
Changed in [b069c7b](https://github.com/OpenImaging/miqa/pull/333/commits/b069c7b8a869904f0127648c41081bf4324b8c13)